libs/application/templates/rental-agreement/src/fields/Summary/OtherFeesSummary.tsx

```
@@ -0,0 +1,241 @@
+            />
+          </GridColumn>
+        </SummaryCardRow>
+      )}
+      {tenantPaysOtherFees &&
+        filterEmptyCostItems(isOtherCostItems).map((item, index) => (
+          <SummaryCardRow
+            editAction={goToScreen}
+            route={route}
+            hasChangeButton={hasChangeButton}
+            isLast={otherCostItems && otherCostItems.length - 1 === index}
+          >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)